tests/dynamic_honey_badger.rs

```diff
@@ -99,11 +102,65 @@ where
         if !input_add && network.nodes.values().all(has_remove) {
             network.input_all(Input::Change(Change::NodeChange(pub_keys_add.clone())));
             input_add = true;
+        } else if !rejoined_node0 {
+            if let Some(join_plan) = network
+                .nodes
+                .values()
+                .flat_map(|node| node.outputs())
+                .filter_map(|batch| match batch.change() {
```

Comment:
There seems to be an `Iterator::find_map` now. That would make the `next()` call unnecessary.